incubator/hnc/internal/kubectl/describe.go

```diff
@@ -67,6 +73,34 @@ var describeCmd = &cobra.Command{
 			fmt.Printf("  No children\n")
 		}
 
+		// Events
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+		events, _ := k8sClient.CoreV1().Events(nnm).List(ctx, metav1.ListOptions{})
+		if events.Size() == 0 {
+			fmt.Printf("  No recent events\n")
+		} else {
+			w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
+			fmt.Fprintln(w, "  Events:\n  Last Seen\tReason\tObject\tMessage")
+			set := make(map[string]bool)
+			// iterate in reverse order so that we only show the latest event if there's an duplicate
+			for i := len(events.Items) - 1; i >= 0; i-- {
+				event := events.Items[i]
+				obj := event.InvolvedObject.Kind + ":" + event.InvolvedObject.Namespace + "/" + event.InvolvedObject.Name
+				if set[event.Reason+obj] {
```

Comment:
Where are we checking only for HNC events?

Reply:
added